system-addon/content-src/components/TopSites/TopSites.jsx

```diff
@@ -80,6 +168,7 @@ const TopSites = props => (<section>
   </ul>
 </section>);
 
-module.exports = connect(state => ({TopSites: state.TopSites}))(TopSites);
+module.exports = connect(state => ({TopSites: state.TopSites}))(TopSitesPerfTimer);
 module.exports._unconnected = TopSites;
```

Comment:
I'm confused.  _unconnected is already set to just TopSites.  What would need to be changed?